### PR TITLE
SLM API doc

### PR DESCRIPTION
**Pull request by urbiz-nr**
NerdGraph doc for SLMs.
* Nav file tweaks
* Added link in summary
* 1st Draft

See DOC-7558 for more details.

### CODE REVIEW

**Review by x8a — APPROVED** [review]
👏🏻👏🏻👏🏻

File: src/content/docs/apis/nerdgraph/examples/nerdgraph-slm.mdx [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+  * Or, for a latency SLI, include a description such as “The proportion of valid requests that were served faster than 0.25s, which is considered to correspond to a good experience”. 
+* **`accountId`**: The ID of the account where the APM service or browser application belongs to, which contains the NRDB data for the SLI/SLO calculations. 
+* **`badEvents.from`**, **`badEvents.where`** 
+  * The NRQL query that defines bad events, `SELECT count(*) FROM badEvents.from WHERE badEvents.where`, requires these `FROM` and `WHERE` clauses.  
+  * If you defined an SLI from valid and bad events, leave the `goodEvents` object empty. 
+* **`goodEvents.from`**, **`goodEvents.where`** 
+  * The NRQL query that defines good events, `SELECT count(*) FROM goodEvents.from WHERE goodEvents.where`, requires these `FROM` and `WHERE` clauses.  
+  * If you defined an SLI from valid and good events, leave the `badEvents` object empty. 
+* **`validEvents.from`**, **`validEvents.where`**
+  * These are the `FROM` and `WHERE` clauses for the NRQL query that defines valid events, which will result in `SELECT (*) FROM validEvents.from WHERE validEvents.where`.
```
Comment (helenapm) [inline src/content/docs/apis/nerdgraph/examples/nerdgraph-slm.mdx:78]: Hi @urbiz-nr ! The query function is count(*) here as well, please. Apologies for introducing this bug on the original draft.

**Review by helenapm — CHANGES_REQUESTED** [review]
Just a small change on a query. Thanks!

File: src/content/docs/apis/nerdgraph/examples/nerdgraph-slm.mdx (hunk 1 above)
Comment (urbiz-nr) [reply src/content/docs/apis/nerdgraph/examples/nerdgraph-slm.mdx:78]: No problem, I'm committing the fix in a sec!

**Pull request merged**